.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
-python: 2.7
+# Currently, Travis only makes Python 3.5 available if it's the version declared here.
+python: 3.5
```

Comment:
Can we do something like [this](https://github.com/hynek/attrs/blob/master/.travis.yml#L9-L32)? I've been meaning to do this, as it gives a better visual within travis too:

Current (from this PR):
![2016-02-15 at 12 13 pm](https://cloud.githubusercontent.com/assets/1191069/13055462/7c1d5b86-d3dd-11e5-907d-ecca35dc9e58.png)

Proposed (from the attrs repo):
![2016-02-15 at 12 12 pm](https://cloud.githubusercontent.com/assets/1191069/13055455/6fe17816-d3dd-11e5-81d7-b1a87f23bf15.png)